packages/pipewire.rb

```diff
@@ -31,6 +32,30 @@ class Pipewire < Package
   depends_on 'vulkan_headers'
   depends_on 'mesa'
 
+  def self.patch
+    case ARCH
+    when 'i686'
+      # Patch from https://gitlab.freedesktop.org/pipewire/pipewire/-/commit/9f53057b51c9d7ce68c240c21b459dc0b7d6acaf
+      # getrandom was introduced to glibc 2.25, and i686 has 2.23.
+      @getrandom_freebsd = <<~'GETRANDOM_FREEBSD_EOF'
+        #include <sys/param.h>
+        #include <fcntl.h>
+        ssize_t getrandom(void *buf, size_t buflen, unsigned int flags) {
+          int fd = open("/dev/random", O_CLOEXEC);
+          if (fd < 0)
+            return -1;
+          ssize_t bytes = read(fd, buf, buflen);
+          close(fd);
+          return bytes;
+        }
+      GETRANDOM_FREEBSD_EOF
+      IO.write('getrandom.c', @getrandom_freebsd)
+      system "sed -i '/random.h/ r getrandom.c' src/pipewire/impl-core.c"
+      system 'grep -4 "ssize_t getrandom" src/pipewire/impl-core.c'
```

Comment:
Not seeing the purpose of this line.

Reply:
That would be debug code trying to figure out why an earlier version wasn't working...  :/